composer.json

```diff
@@ -16,14 +16,14 @@
         "sort-packages": true
     },
     "require": {
-        "php": "^7.2",
+        "php": "^7.2|^8.0",
         "ext-pdo": "*",
         "composer/package-versions-deprecated": "^1.8",
         "doctrine/annotations": "^1.11.1",
         "doctrine/cache": "^1.9.1",
         "doctrine/collections": "^1.5",
         "doctrine/common": "^3.0",
-        "doctrine/dbal": "^2.10.0",
+        "doctrine/dbal": "^2.12.0",
```

Comment:
Why are we bumping this? Can't we let Composer figure things out and restore the 7.2 job?

Reply:
@greg0ire because PHP 8 requires 2.12

Reply:
I plan to merge the PHP 7.1 support for DBAL 2.12 in the next days, so that means we can add them in ORM as well. But that can be seperate: https://github.com/doctrine/dbal/pull/4386